types/nightwatch/index.d.ts

```diff
@@ -6412,4 +6412,25 @@ export interface PageObjectModel {
     props?: any;
 }
 
-export const Nightwatch: NightwatchClient & Nightwatch;
+export function cli(callback: any): Nightwatch;
+export function client(settings: NightwatchOptions, reporter?: any, argv?: {}): Nightwatch;
+export function createClient({
+    headless,
+    silent,
+    output,
+    useAsync,
+    env,
+    timeout,
+    parallel,
+    reporter,
+    browserName,
+    globals,
+    devtools,
+    debug,
+    enable_global_apis,
+    config,
+}: CreateClientParams): Nightwatch;
+export function CliRunner(argv?: {}): Nightwatch;
+export function initClient(opts: any): Nightwatch;
+export function runner(argv?: {}, done?: () => void, settings?: {}): Nightwatch;
+export function runTests(testSource: string | string[], settings?: any, ...args: any[]): any;
```

Comment:
Apologies for the late review, but this whole chunk of typings is actually redundant. Also, I presume that `import * as Nightwatch from 'nightwatch';` is not actually the correct way of using the programmatic API?

I tried using `export =` but that also does not seem to work and since those using Nightwatch with Typescript would actually be importing the module using `import Nightwatch from 'nightwatch'` and not `import Nightwatch = require('nightwatch')`, `export default` seems safe to use to me. (Ref for `export =`: https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require)

So, can you please change the above back to:
```js
declare const _default: Nightwatch;
export default _default;
```

Reply:
Ok I changed it back.